Workflows/owlbot.py

```diff
@@ -79,20 +68,6 @@
     r"\$transportConfig, and any \$serviceAddress",
     r"$transportConfig, and any `$apiEndpoint`")
 
-# fix year
-s.replace(
-    '**/Gapic/*GapicClient.php',
-    r'Copyright \d{4}',
-    'Copyright 2020')
-s.replace(
-    '**/V1beta/*Client.php',
-    r'Copyright \d{4}',
-    'Copyright 2020')
-s.replace(
-    'tests/**/V1beta/*Test.php',
-    r'Copyright \d{4}',
-    'Copyright 2020')
-
 # Change the wording for the deprecation warning.
 s.replace(
     'src/*/*_*.php',
```

Comment:
`Workflows` is pre-1.0, so we could remove all these BC proto files... I think we might as well do this for all these libraries while we are migrating to owlbot

Reply:
@bshaffer Done. PTAL